tck/src/main/java/com/akkaserverless/javasdk/tck/model/valueentity/ValueEntityTckModelEntity.java

```diff
@@ -44,38 +45,40 @@ public ValueEntityTckModelEntity(Context context) {
   }
 
   @CommandHandler
-  public Reply<Response> process(Request request, CommandContext<Persisted> context) {
-    Reply<Response> reply = null;
-    List<SideEffect> e = new ArrayList<>();
-    for (RequestAction action : request.getActionsList()) {
-      switch (action.getActionCase()) {
-        case UPDATE:
-          state = action.getUpdate().getValue();
-          context.updateState(Persisted.newBuilder().setValue(state).build());
-          break;
-        case DELETE:
-          context.deleteState();
-          state = "";
-          break;
-        case FORWARD:
-          reply = Reply.forward(serviceTwoRequest(action.getForward().getId()));
-          break;
-        case EFFECT:
-          Effect effect = action.getEffect();
-          e.add(SideEffect.of(serviceTwoRequest(effect.getId()), effect.getSynchronous()));
-          break;
-        case FAIL:
-          reply = Reply.failure(action.getFail().getMessage());
-          break;
-      }
-    }
-    if (reply == null) {
-      reply = Reply.message(Response.newBuilder().setMessage(state).build());
-    }
-    return reply.addSideEffects(e);
+  public Effect<Response> process(Request request, CommandContext<Persisted> context) {
+    // FIXME the effect API doesn't support all combinations, and that might be fine?
+    //    Effect<Response> reply = null;
+    //    List<SideEffect> e = new ArrayList<>();
+    //    String newState = state;
+    //    for (RequestAction action : request.getActionsList()) {
+    //      switch (action.getActionCase()) {
+    //        case UPDATE:
+    //          newState = action.getUpdate().getValue();
+    //          break;
+    //        case DELETE:
+    //          newState = "";
+    //          break;
+    //        case FORWARD:
+    //          reply = effects().forward(serviceTwoRequest(action.getForward().getId()));
+    //          break;
+    //        case EFFECT:
+    //          com.akkaserverless.tck.model.ValueEntity.Effect effect = action.getEffect();
+    //          e.add(SideEffect.of(serviceTwoRequest(effect.getId()), effect.getSynchronous()));
+    //          break;
+    //        case FAIL:
+    //          return effects().error(action.getFail().getMessage());
+    //      }
+    //    }
+    //    effects().updateState(Persisted.newBuilder().setValue(state).build());
```

Comment:
I remember this was tricky with reply in the JS sdk as well, perhaps what I did there to only keep the last reply/effect around could be useful to look at as inspiration? https://github.com/lightbend/akkaserverless-javascript-sdk/blob/main/tck/value-entity.js#L57